File: src/main/java/com/naren/sms/request/Subject.java
package com.naren.sms.request;

import java.sql.Timestamp;

@lombok.Getter
@lombok.Setter
@lombok.ToString
public class Subject {

    Long subjectId;

    private String subjectName;

    private String subjectDescription;

    private Boolean deleted = false;

    private Timestamp createdDate;

    private Timestamp updatedDate;

    private String createdBy;

    private String updatedBy;

}

File: web/src/pages/subjects/subject.ts
export class Subject {
    subjectId?;
    subjectName?;
    subjectDescription?;
    deleted: boolean;
    createdBy: string;
    updatedBy: string;

    constructor(obj: any) {
        this.subjectId = obj.subjectId;
        this.subjectName = obj.subjectName;
        this.subjectDescription = obj.subjectDescription;
        this.deleted = obj.deleted;
        this.createdBy = obj.createdBy;
        this.updatedBy = obj.updatedBy;
    }
}